sql/main_test.go

```diff
@@ -40,6 +41,12 @@ func init() {
 //go:generate ../util/leaktest/add-leaktest.sh *_test.go
 
 func TestMain(m *testing.M) {
+	// All our tests are written assuming UTC.
```

Comment:
Instead of simulating a case where the server's time zone is set to UTC, we should simulate a case where the server's local time is definitely *not* UTC, and set the session time zone to UTC for every test. This can come in a separate change, though, if there are a lot of tests that would have to be touched.